rdkit/Dbase/DbInfo.py

```diff
@@ -120,13 +122,13 @@ def GetColumnInfoFromCursor(cursor):
     for i, v in enumerate(r):
       cName = cursor.description[i][0]
       typ = type(v)
-      if typ == str or (PY2 and typ == unicode):
+      if typ == str or (PY2 and typ == unicode):  # @UndefinedVariable
```

Comment:
Seems like this would be cleaner as `typ in six.string_types` with an import of `rdkit.six` somewhere above.